rcore/buttons.c

```diff
@@ -373,6 +373,8 @@ void button_raw_click_subscribe(ButtonId button_id, ClickHandler down_handler, C
     ButtonHolder *holder = _button_holders[button_id]; // get the button
     holder->click_config.raw.up_handler = up_handler;
     holder->click_config.raw.down_handler = down_handler;
+    if (context != NULL)
+        holder->click_config.context = context;
```

Comment:
This seems like it will leak old context data -- should it always be set (even if to NULL)?

Reply:
I don't think old context values might leak as before a call to the click config provider, all context values for the buttons are set to the click config provider context. 
For the always be set thing, the docu states that `If this context is not NULL, it will override the general context`, so if it is NULL I assume it should not override.

Reply:
Seems ok to me. the context does seems to be set appropriately. Merge